platform/src/main/java/org/hillview/table/api/IColumn.java

```diff
@@ -118,7 +118,12 @@ default String getName() {
     default ContentsKind getKind() { return this.getDescription().kind;}
 
     /**
-     * @return A hash code for the item in the rowIndex. Exception if it is missing.
+     * @return A 64 bit hash code for the item in the rowIndex.  xors the item with the seed and then runs xxhashing
```

Comment:
That's a good idea and a trivial change to make. I could do it in the future if we feel it is needed. One downside is that a reference to the XXHash would pass as part of the sketch, isn't there an overhead of serialization?